src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -205,25 +205,29 @@ def _get_request_origin_and_config(
 
     Additionally validate that the experience config history is valid if supplied.
     """
-    privacy_experience_history: Optional[PrivacyExperienceHistory] = None
-    experience_config_id: Optional[str] = None
-    if data.privacy_experience_history_id:
-        privacy_experience_history = PrivacyExperienceHistory.get(
-            db=db, object_id=data.privacy_experience_history_id
+    privacy_experience: Optional[PrivacyExperience] = None
+    experience_config_history_id: Optional[str] = None
+    if data.privacy_experience_id:
+        privacy_experience = PrivacyExperience.get(
+            db=db, object_id=data.privacy_experience_id
         )
-        if not privacy_experience_history:
+        if not privacy_experience:
             raise HTTPException(
                 status_code=HTTP_404_NOT_FOUND,
-                detail=f"Privacy Experience History '{data.privacy_experience_history_id}' not found.",
+                detail=f"Privacy Experience '{data.privacy_experience_id}' not found.",
+            )
+        experience_config_id = privacy_experience.experience_config_id
+        if experience_config_id:
+            experience_config_history_id = (
+                privacy_experience.experience_config.experience_config_history_id
```

Comment:
When saving privacy preferences, I now need the frontend to pass in the "privacy_experience_id" instead of the "privacy_experience_history_id" (which no longer exists).  We can't derive the experience from the notice alone, so this extra field lets us look up the experience config id that would have been used to surface the notices to the customer and save them for record keeping.